content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -1,8 +1,136 @@
 # DVC Files and Directories
 
 Once initialized in a <abbr>project</abbr>, DVC populates its installation
-directory (`.dvc/`) with the internal files and directories needed for DVC
-operation:
+directory (`.dvc/`) with the
+[internal directories and files](#internal-directories-and-files) needed for DVC
+operation.
+
+Additionally, there are two special files created by certain
+[DVC commands](/doc/command-reference):
+
+- Files ending with the `.dvc` extension are basic placeholders to version data
+  files and directories. A <abbr>DVC project</abbr> can have multiple
+  [`.dvc` files](#dvc-files).
+- The [`dvc.yaml` file](#dvcyaml-file) or _pipeline(s) file_ specifies stages
+  that form the pipeline(s) of a project, and their connections (_dependency
+  graph_ or DAG).
+
+Both use human-friendly YAML schemas, described below. We encourage you to get
+familiar with them so you may edit them freely, as needed. Both type of files
+should be versioned with Git (for Git-enabled <abbr>repositories</abbr>).
+
+## .dvc files
+
+When you add a file or directory to a <abbr>DVC project</abbr> with `dvc add` or
+`dvc import`, a `.dvc` file is created based on the data file name (e.g.
+`data.xml.dvc`). These files contain the basic information needed to track the
+data with DVC.
+
+They use a simple YAML format that can be easily written or altered manually.
+Here is a sample:
+
+```yaml
+outs:
```

Comment:
Btw, this is correct for `dvc add`ed files, but for `dvc import`ed (or import-url) it is a bit more complex, same as it was before.

Reply:
Good point. I've added the deps section for .dvc files as well now. See bdab8aee

Please lmk if the latest version has any other errors or missing info. Thanks @efiop